pkg/command/context.go

```diff
@@ -46,35 +46,38 @@ import (
 )
 
 var (
-	stderrOnly, forceCSP, staging, onlyCurrent, skipTLSVerify                                           bool
-	ctxName, endpoint, apiToken, kubeConfig, kubeContext, getOutputFmt, endpointCACertPath, contextType string
-	projectStr, spaceStr                                                                                string
+	stderrOnly, forceCSP, staging, onlyCurrent, skipTLSVerify                              bool
+	ctxName, endpoint, apiToken, kubeConfig, kubeContext, getOutputFmt, endpointCACertPath string
+	projectStr, spaceStr                                                                   string
 )
 
 const (
 	knownGlobalHost    = "cloud.vmware.com"
 	apiTokenType       = "api-token"
 	defaultTAEEndpoint = "https://api.tanzu.cloud.vmware.com"
 
-	contextNotExistsForTarget      = "The provided context %v does not exist or is not active for the given target %v"
-	noActiveContextExistsForTarget = "There is no active context for the given target %v"
-	contextNotActiveOrNotExists    = "The provided context %v is not active or does not exist"
-	contextForTargetSetInactive    = "The context %v for the target %v has been set as inactive"
-	deactivatingPlugin             = "Deactivating plugin '%v:%v' for context '%v'"
+	contextNotExistsForContextType      = "The provided context '%v' does not exist or is not active for the given context type '%v'"
+	noActiveContextExistsForContextType = "There is no active context for the given context type '%v'"
+	contextNotActiveOrNotExists         = "The provided context '%v' is not active or does not exist"
+	contextForContextTypeSetInactive    = "The context '%v' of type '%v' has been set as inactive"
+	deactivatingPlugin                  = "Deactivating plugin '%v:%v' for context '%v'"
 
-	invalidTarget = "invalid target specified. Please specify a correct value for the `--target/-t` flag from 'kubernetes[k8s]/mission-control[tmc]/application-engine[tae]"
+	invalidTarget      = "invalid target specified. Please specify a correct value for the `--target/-t` flag from 'kubernetes[k8s]/mission-control[tmc]/application-engine[tae]"
```

Comment:
There is single quote missing at the end of the list of context types (pre-existing)